python/export_util.py

```diff
@@ -108,6 +111,262 @@ def get_graph(ctx: mgp.ProcCtx) -> List[Union[Node, Relationship]]:
     return nodes + relationships
 
 
+def get_graph_from_lists(node_list: list, relationship_list: list):
+
+    graph = list()
+    node_properties = list()
+    node_prop_set = set()
+    relationship_properties = list()
+    relationship_prop_set = set()
+
+
+    for node in node_list:
+        for prop in node.properties:
+            if not prop in node_prop_set:
+                node_properties.append(prop)
+                node_prop_set.add(prop)
+        node_properties.sort()
+        graph.append(
+
+            Node(node.id, node.labels, node.properties)
+
+        )
+
+    for relationship in relationship_list:
+        for prop in relationship.properties:
+            if not prop in relationship_prop_set:
+                relationship_properties.append(prop)
+                relationship_prop_set.add(prop)
+        relationship_properties.sort()
+
+        graph.append(
+
+            Relationship(
+
+                relationship.to_vertex.id,
+
+                relationship.id,
+
+                relationship.type.name,
+
+                relationship.properties,
+
+                relationship.from_vertex.id,
+
+            )
+
+        )
+
+    return graph, node_properties, relationship_properties
+
+def csv_header(node_properties: list, relationship_properties: list) -> list:
+    """
+    This function creates the header for csv file 
+    """
+    header = ["_id","_labels"]
+
+    for prop in node_properties:
+        header.append(prop)
+
+    header.extend(["_start", "_end", "_type"])
+
+    for prop in relationship_properties:
+        header.append(prop)
+
+    return header
+
+def csv_data_list(graph: list, node_properties: list, relationship_properties: list) -> list:
+
+    """
+    Function that parses graph into a data_list appropriate for csv writing
+    """
+    data_list = []
+    for element in graph:
+        write_list = []
+        IsNode = isinstance(element, Node)
+        
+        #id and labels
+        if(IsNode):
+            write_list.extend([element.id, "".join(":" + label.name for label in element.labels)])
+        else:
+            write_list.extend(["", ""])
+
+        #node_properties
+        for prop in node_properties:
+            if prop in element.properties and IsNode:
+                if isinstance(element.properties[prop], (set, list, tuple, map)):
+                    write_list.append(js.dumps(element.properties[prop]))
+                else:   
+                    if isinstance(element.properties[prop],timedelta):
+                        write_list.append(convert_to_isoformat(element.properties[prop]))
+                    else:
+                        write_list.append(element.properties[prop])
+            else:
+                write_list.append("")
```

Comment:
I think a function wouldnt be a good idea, because write_list is a list, which woul dneed to be passed as an argument for properties part, so we would copy list every time we called the function, and the we would have to append its contents to write_list, also, i dont understand what exactly you mean by fewer indents?